corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -194,3 +194,8 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if "pretty" in request.GET:
         kw["indent"] = 2
     return HttpResponse(json.dumps(data, **kw))
+
+
+@require_GET
```

Comment:
did you intentionally not have this require login? I don't know how the session timeout stuff works, but it seems like forcing authentication couldn't hurt unless you were just worried about performance